ui/src/components/audio_recorder_flow.jsx

```diff
@@ -93,8 +93,8 @@ export default React.createClass({
   },
 
   onDoneCapture(blob:Blob) {
-    this.props.onLogMessage('message_popup_audio_on_done_capture');
     var downloadUrl = URL.createObjectURL(blob);
+    this.props.onLogMessage('message_popup_audio_on_done_capture', {'blobUrl': downloadUrl});
```

Comment:
This is a great solution for the playtest!  We should revisit this later though, since `onLogMessage` is about logging messages to the server, and these `blobUrl`s will have no meaning outside of this particular browser.